contracts/connectors/1proto/helpers.sol

```diff
@@ -0,0 +1,39 @@
+pragma solidity ^0.7.0;
+
+import { TokenInterface } from "../../common/interfaces.sol";
+import { DSMath } from "../../common/math.sol";
+import { Basic } from "../../common/basic.sol";
+import { TokenInterface, OneProtoMappingInterface } from "./interface.sol";
+
+abstract contract Helpers is DSMath, Basic {
+    /**
+     * @dev 1proto mapping Address
+     */
+    OneProtoMappingInterface constant internal oneProtoMapping = OneProtoMappingInterface(0x8d0287AFa7755BB5f2eFe686AA8d4F0A7BC4AE7F);
+
+    /**
+     * @dev Return 1proto Address
+     */
+    function getOneProtoAddress() internal virtual view returns (address payable) {
+        return payable(oneProtoMapping.oneProtoAddress());
```

Comment:
@mubaris I don't think 1inch is going to update the 1proto contract address. As it's deprecated. We can directly use the 1proto.eth address here.

Reply:
Updated